### PR TITLE
Specify component's displayName

### PR DESCRIPTION
**Pull request by leshakoss**
For better debugging:
screen shot 2017-07-04 at 13 36 17
screen shot 2017-07-04 at 13 38 03

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function decss (h, style, defaultProps) {
   var blocks = getBlocks(style)
 
   return Object.keys(blocks).reduce(function (acc, blockName) {
-    acc[blockName] = function (props) {
+    const component = function (props) {
```
Comment (kossnocorp) [inline index.js]: Please use `var` for compatibility sake.

Comment (leshakoss) [inline index.js]: There's `const` on the next line as well. Should I fix it while I'm here?

Comment (leshakoss) [inline index.js]: @kossnocorp Done!

**Comment by kossnocorp** [comment]
Thanks!

**Pull request merged**